core/feature-manager/org.wso2.carbon.feature.mgt.core/pom.xml

```diff
@@ -125,6 +125,23 @@
             <groupId>org.wso2.eclipse.equinox</groupId>
             <artifactId>org.eclipse.equinox.simpleconfigurator</artifactId>
         </dependency>
+	<dependency>
+	    <groupId>org.osgi</groupId>
```

Comment:
can't we use 'org.wso2.eclipse.osgi'

Reply:
it doesn't have required dependencies with group id org.osgi.service.component.annotations .